### PR TITLE
Add Irajain7 hometown  : india

### LINKED ISSUE #21: Create a new file for irajain7 Hometown
### INDIA
- [x] Create  a branch
- [x] Add commits
- [x] Open a PR
- [x] Discussion on PR
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by irajain7**
resolves #21

### CODE REVIEW

File: irajain7-India.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# India
+
+### Places to go
+- Delhi
+- Taj Mahal
+
```
Comment (snikolaev) [inline irajain7-India.md]: @irajain7  Where is Taj Mahal? How to get there?

Comment (irajain7) [reply irajain7-India.md]: @snikolaev  This should give all info 🎉 
http://whc.unesco.org/en/list/252

**Comment by mathieu6969** [comment]
👍 looks good!

**Comment by simonliu9g** [comment]
👍 looks good!

**Pull request merged**